tests/ltp/patches/mount.patch

```diff
@@ -0,0 +1,409 @@
+diff --git a/testcases/kernel/syscalls/mount/mount01.c b/testcases/kernel/syscalls/mount/mount01.c
+index 1d902ba89..86ffd95c4 100644
+--- a/testcases/kernel/syscalls/mount/mount01.c
++++ b/testcases/kernel/syscalls/mount/mount01.c
+@@ -36,8 +36,13 @@ int TST_TOTAL = 1;
+ #define DIR_MODE (S_IRWXU | S_IRUSR | S_IXUSR | S_IRGRP | S_IXGRP)
+ #define MNTPOINT "mntpoint"
+ 
+-static const char *device;
+-static const char *fs_type;
++/*
++	patch to use root file system as sgx has 32 MB limit in using in 
```

Comment:
This comment is not accurate, it's not a limitation of SGX but rather our choice of setting the default LKL kernel memory to 32M.